lib/hyrax/transactions/apply_change_set.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+require 'hyrax/transactions/transaction'
+
+module Hyrax
+  module Transactions
+    ##
+    # Applies and saves a ChangeSet.
+    #
+    # @since 3.0.0
+    #
+    # @example Applying a ChangeSet to a Work
+    #   work = Hyrax::Work.new
+    #   change_set = Hyrax::ChangeSet.for(work)
+    #   change_set.title = ['Comet in Moominland']
+    #
+    #   transaction = Hyrax::Transactions::ApplyChangeSet.new
+    #   persisted   = transaction.call(change_set).value_or(:error)
```

Comment:
I'm assuming that the `.value_or(:error)` is something from the Dry world? It reads a bit awkwardly.

Reply:
yeah, it's a `Dry::Monad` thing